### PR TITLE
Fix new tags not being plural

### PR DESCRIPTION
**Pull request by TelepathicGrunt**
This is entirely my fault. I should've caught the typoes. Gave them the same treatment as the tool tags of deprecating the typoed stuff

Fixes https://github.com/FabricMC/fabric/pull/3807

Matches https://github.com/neoforged/NeoForge/pull/1030

### CODE REVIEW

File: fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/ConventionalItemTags.java [hunk 1]
```diff
@@ -173,12 +173,12 @@ private ConventionalItemTags() {
 	 * Block tag equivalent is {@link BlockTags#SHULKER_BOXES}.
 	 */
 	public static final TagKey<Item> SHULKER_BOXES = register("shulker_boxes");
-	public static final TagKey<Item> GLAZED_TERRACOTTA = register("glazed_terracotta");
-	public static final TagKey<Item> CONCRETE = register("concrete");
+	public static final TagKey<Item> GLAZED_TERRACOTTAS = register("glazed_terracottas");
+	public static final TagKey<Item> CONCRETES = register("concretes");
 	/**
 	 * Block tag equivalent is {@link BlockTags#CONCRETE_POWDER}.
```
Comment (haykam821) [inline fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/ConventionalItemTags.java]:
```suggestion
	 * Block tag equivalent is {@link BlockTags#CONCRETE_POWDERS}.
```

File: fabric-convention-tags-v2/src/datagen/java/net/fabricmc/fabric/impl/tag/convention/datagen/generators/EnglishTagLangGenerator.java [hunk 2]
```diff
@@ -194,10 +196,13 @@ public void generateTranslations(RegistryWrapper.WrapperLookup registryLookup, T
 		translationBuilder.add(ConventionalItemTags.WOODEN_CHESTS, "Wooden Chests");
 		translationBuilder.add(ConventionalItemTags.GLASS_BLOCKS, "Glass Blocks");
 		translationBuilder.add(ConventionalItemTags.GLASS_PANES, "Glass Panes");
-		translationBuilder.add(ConventionalItemTags.GLAZED_TERRACOTTA, "Glaze Terracotta");
 		translationBuilder.add(ConventionalItemTags.SHULKER_BOXES, "Shulker Boxes");
+		translationBuilder.add(ConventionalItemTags.GLAZED_TERRACOTTAS, "Glaze Terracottas");
```
Comment (haykam821) [inline fabric-convention-tags-v2/src/datagen/java/net/fabricmc/fabric/impl/tag/convention/datagen/generators/EnglishTagLangGenerator.java]:
```suggestion
		translationBuilder.add(ConventionalItemTags.GLAZED_TERRACOTTAS, "Glazed Terracottas");
```

File: fabric-convention-tags-v2/src/datagen/java/net/fabricmc/fabric/impl/tag/convention/datagen/generators/EnglishTagLangGenerator.java [hunk 3]
```diff
@@ -194,10 +196,13 @@ public void generateTranslations(RegistryWrapper.WrapperLookup registryLookup, T
...
+		translationBuilder.add(ConventionalItemTags.CONCRETES, "Concretes");
+		translationBuilder.add(ConventionalItemTags.CONCRETE_POWDERS, "Concrete Powders");
 		translationBuilder.add(ConventionalItemTags.CONCRETE, "Concrete");
 		translationBuilder.add(ConventionalItemTags.CONCRETE_POWDER, "Concrete Powder");
+		translationBuilder.add(ConventionalItemTags.GLAZED_TERRACOTTA, "Glaze Terracotta");
```
Comment (haykam821) [inline fabric-convention-tags-v2/src/datagen/java/net/fabricmc/fabric/impl/tag/convention/datagen/generators/EnglishTagLangGenerator.java]:
```suggestion
		translationBuilder.add(ConventionalItemTags.GLAZED_TERRACOTTA, "Glazed Terracotta");
```

File: fabric-convention-tags-v2/src/datagen/java/net/fabricmc/fabric/impl/tag/convention/datagen/generators/EnglishTagLangGenerator.java [hunk 4]
```diff
@@ -50,8 +50,10 @@ public void generateTranslations(RegistryWrapper.WrapperLookup registryLookup, T
 		translationBuilder.add(ConventionalBlockTags.WOODEN_CHESTS, "Wooden Chests");
 		translationBuilder.add(ConventionalBlockTags.GLASS_BLOCKS, "Glass Blocks");
 		translationBuilder.add(ConventionalBlockTags.GLASS_PANES, "Glass Panes");
-		translationBuilder.add(ConventionalBlockTags.GLAZED_TERRACOTTA, "Glaze Terracotta");
 		translationBuilder.add(ConventionalBlockTags.SHULKER_BOXES, "Shulker Boxes");
+		translationBuilder.add(ConventionalBlockTags.GLAZED_TERRACOTTAS, "Glaze Terracottas");
```
Comment (haykam821) [inline fabric-convention-tags-v2/src/datagen/java/net/fabricmc/fabric/impl/tag/convention/datagen/generators/EnglishTagLangGenerator.java]:
```suggestion
		translationBuilder.add(ConventionalBlockTags.GLAZED_TERRACOTTAS, "Glazed Terracottas");
```

File: fabric-convention-tags-v2/src/datagen/java/net/fabricmc/fabric/impl/tag/convention/datagen/generators/EnglishTagLangGenerator.java [hunk 5]
```diff
@@ -50,8 +50,10 @@ public void generateTranslations(RegistryWrapper.WrapperLookup registryLookup, T
...
+		translationBuilder.add(ConventionalBlockTags.CONCRETES, "Concretes");
+		translationBuilder.add(ConventionalBlockTags.GLAZED_TERRACOTTA, "Glaze Terracotta");
```
Comment (haykam821) [inline fabric-convention-tags-v2/src/datagen/java/net/fabricmc/fabric/impl/tag/convention/datagen/generators/EnglishTagLangGenerator.java]:
```suggestion
		translationBuilder.add(ConventionalBlockTags.GLAZED_TERRACOTTA, "Glazed Terracotta");
```

**Review by haykam821 — CHANGES_REQUESTED** [review]

File: fabric-convention-tags-v2/src/datagen/java/net/fabricmc/fabric/impl/tag/convention/datagen/generators/EnglishTagLangGenerator.java (hunk 5 above)
Comment (modmuss50) [reply fabric-convention-tags-v2/src/datagen/java/net/fabricmc/fabric/impl/tag/convention/datagen/generators/EnglishTagLangGenerator.java]: Resources need re-generating after applying these changes.

Comment (TelepathicGrunt) [reply fabric-convention-tags-v2/src/datagen/java/net/fabricmc/fabric/impl/tag/convention/datagen/generators/EnglishTagLangGenerator.java]: done in b5893fff445073752c88dc4cf69956acf19840a6

**Comment by TelepathicGrunt** [comment]
@modmuss50 any concerns or is this good for approval?

**Comment by modmuss50** [comment]
> @modmuss50 any concerns or is this good for approval?

Sorry, been a bit busy (IRL things, and porting my mod to 1.21 :D ). Ill try and take ASAP.

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**